### DISCUSSION TITLE
visualizer exception after updating scmlv

### DISCUSSION THREAD

**Discussion by snippet2021** [yasserfarouk/scml-vis#7] [General]
I have trouble after updating scml-vis today.
I use oneShotworld. I run myagent.py and then I want to see visualizer.
When I select the tournament and compile it ,I got this exception.

> StreamlitAPIException: expander() is not a valid Streamlit command.
> Traceback:
> File "/home/**(mydir)**/.venv/lib/python3.8/site-packages/scml_vis/presenter.py", line 1323, in <module>
>     main(folder)
> File "/home/**(mydir)**/.venv/lib/python3.8/site-packages/scml_vis/presenter.py", line 119, in main
>     tournament_expander = st.sidebar.expander("Tournament Selection")

It is also output to the command line as shown below.

> Tournament 20220428H222750051930TFI-stage-0001 [0 of 1]
>         World 0000Fi___main__.MyAgentzbhF.00 [0 of 3]
>         World 0000Fi_scml.oneshot.agents.random.RandomOneShotAgentZRRH.00 [1 of 3]
>         World 0000Fi_scml.oneshot.agents.random.SyncRandomOneShotAgentcVZw.00 [2 of 3]
>         Did not find neg_stats
>         Did not find breaches

Could you help me with this?

**Comment by yasserfarouk** [comment]
This is an issue with streamlit version

please try 
```path
python -m pip install -U streamlit
```

That is a temporary fix until we update the version in the next patch.

**Reply by snippet2021** [reply]
After updating streamlit, it works!
Thank you very much for your support!!

**Reply by yasserfarouk** [reply]
Great
